packages/replay/src/coreHandlers/addBreadcrumbEvent.ts

```diff
@@ -12,7 +12,7 @@ export function addBreadcrumbEvent(replay: ReplayContainer, breadcrumb: Breadcru
     return;
   }
 
-  if (breadcrumb.category === 'ui.click') {
+  if (['ui.click', 'ui.input'].includes(breadcrumb.category || '')) {
```

Comment:
l: IMHO we do not need the `|| ''` part, it should still work with `includes(undefined)` - we can safe some bytes by doing `includes(breadcrumb.category as string)` (to satisfy TS)?